packages/x-components/tests/unit/main-scroll.spec.ts

```diff
@@ -141,18 +139,17 @@ describe('testing MainScroll component', () => {
         userScrolledToElementSpy().should('have.been.calledWith', 'item-5');
       });
 
-      it('ignores the first pixels of scroll', () => {
+      it.only('ignores the first element of scroll', () => {
```

Comment:
Please remove the `only` keyword, nothing is going to happen, believe in yourself!